sdk/purview/azure-purview-sharing/CHANGELOG.md

```diff
@@ -1,17 +1,5 @@
 # Release History
 
-## 1.0.0b3 (2023-07-14)
-
-### Bugs Fixed
-
-- Updated broken link in samples README to point to Azure/azure-python-sdk instead of developer fork
-
-### Other Changes
-
-- Migrated test recordings to the azure-sdk-assets repo
-- Added more sanitizers to conftest.py
-
-
 ## 1.0.0b3 (2023-06-29)
```

Comment:
Just want to double-check, it looks like azure-purview-sharing released on 2023-07-18. Is this right? 

https://pypi.org/project/azure-purview-sharing/#history

Reply:
Oh, yep, that's right. Sorry about that, thank you for catching it!